.gitignore

```diff
@@ -16,3 +16,4 @@ packages/dev/v2-test-deps
 dist
 public
 storybook-static
+junit.xml
```

Comment:
Why is this change part of this pr?

Reply:
happened to notice it and didn't want to commit it